packages/react-native/scripts/cocoapods/utils.rb

```diff
@@ -396,6 +396,42 @@ def self.set_imagemanager_search_path(target_installation_result)
         ])
     end
 
+    def self.get_plist_paths_from(user_project)
+        info_plists = user_project
+          .files
+          .select { |p|
+            p.name&.end_with?('Info.plist')
+          }
+        return info_plists
+      end
+
+    def self.update_ats_in_plist(plistPaths, parent)
+        plistPaths.each do |plistPath|
+            fullPlistPath = File.join(parent, plistPath.path)
```

Comment:
This would only be accurate if you trace the join the paths of `PBXFileReference.path | PBXFileReference.name` with every parent `PBXGroup.path` according to `sourceTree`. Iirc cocoapods has some utility method for getting the real file path for a file.